src/@next/Tooltip/TooltipStyle.ts

```diff
@@ -131,6 +131,42 @@ export const StyledTooltip = styled.div<{ zIndex?: number }>`
     bottom: ${nonCentralArrowMargin};
   }
 
+  &[data-clickable='true'] {
+    animation: fade-in-tooltip 400ms ease-in forwards;
```

Comment:
[tooltip.webm](https://github.com/glints-dev/glints-aries/assets/12844836/71f1f28e-2f60-434f-a75a-7e8081dead9d)
Not sure what is causing this movement of the tooltip? Could it be due to the animation?

Reply:
I thought you intended that by having the transform: scale(...) on the animation CSS you give me? is that not intended?

Reply:
nope haha, okay I think maybe we should just get rid of transform:scale for now

Reply:
or did you mean to actually enlarge the tooltip or make it smaller by that scale instead of moving it?

Reply:
okayy will get rid of it first now

Reply:
@michac789 yes, coz we are not setting transform to `translate()` but `scale()` only, so it should have remained its position.